packages/create/src/index.ts

```diff
@@ -88,7 +88,8 @@ export async function createLightNode(
   const store = wakuStore(options);
```

Comment:
You have not removed the `CreateOptions` definition in this file. I expect it to be replaced with `CreateProtocolOPtions` from @waku/interfaces